include/hcl/concurrent/unordered_map/Block_chain_memory.h

```diff
@@ -0,0 +1,280 @@
+#ifndef __BLOCK_CHAIN_
```

Comment:
why is this datastructure under unordered map? 

Again if its not extending container move it to include/hcl/base/containers/
fix file name the conversion is not followed correctly,

Reply:
It is a concurrent implementation and not a sequential implementation. Do you want me to change its name?